File: code_examples/3-Structured_Problem_Solving/2-Constants/C/main.c
// Copyright (c) 2020 Mr. Coxall All rights reserved.
//
// Created by: Mr. Coxall
// Created on: Sep 2020
// This program shows declaring constants

#include <stdio.h>

int main() {
    // constant definition
    const int ROOM_NUMBER = 212;
    const float HST = 0.13;
    const char COUNTRY[6] = "Canada";

    printf("Room: %d\n", ROOM_NUMBER);
    printf("HST %.2f%%\n", HST);
    printf("%s\n", COUNTRY);

    printf("\nDone.\n");
    return 0;
}

File: code_examples/3-Structured_Problem_Solving/2-Constants/Go/main.go
/**
 * Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program shows declaring constants
 */

package main

import "fmt"

func main() {
	// constant definition
	const ROOM int = 212
	const HST float64 = 0.13
	const COUNTRY string = "Canada"

	fmt.Println("Room:", ROOM)
	fmt.Println(HST*100, "%")
	fmt.Println(COUNTRY)

	fmt.Println("\nDone.")
}
